sdk/data/ci.yml

```diff
@@ -14,3 +14,7 @@ stages:
   parameters:
     ServiceDirectory: 'data'
     RunTests: true
+
```

Comment:
@benbp shouldn't we include the tests in the common client yml?

Reply:
@weshaggard they are included right below this comment. The `RunTests` parameter gets defaulted to true in the live test stage/job.

```
- template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
  parameters:
    ServiceDirectory: 'tables'
```

Reply:
Oh you mean not have archetype-sdk-tests.yml? Yeah that's a fair point. Let me look into that.

Reply:
I'll refactor after this merge so I don't block tables.

Reply:
https://github.com/Azure/azure-sdk-for-go/pull/15456